nbs/examples/ElectricityLoadForecasting.ipynb

```diff
@@ -0,0 +1,3625 @@
+{
```

Comment:
<p>Remove extra word: We observe that the time required for <code>Prophet</code> <strong><s>es</s></strong> to perform the adjustment and predict is greater than <code>MSTL</code>. Let's look at the forecasts produced by <code>Prophet</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/295/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='65'/>